package TryCatchFinally.customExceptionExample;

public class IntegerOutOfRangeException extends Exception {
}
